api/models/userDao.js

```diff
@@ -37,8 +38,33 @@ const createUser = async (name, email, password, gender, address) => {
     );
 };
 
+const getPointByUserId = async (userId) => {
+    const point = await AppDataSource.query(
+        `SELECT
+            point
+        FROM users
+        WHERE id = ?
+        `,
+        [userId]
+    );
+    return point[0];
+};
+
+const updatePointByUserId = async (point, userId) => {
+    await AppDataSource.query(
```

Comment:
```suggestion
    return await AppDataSource.query(
```
해당 method에서 쿼리 실행 후 더이상 처리할 로직이 없다면 return으로 함수를 마무리해주세요.

추가로 update, delete, create와 관련된 dao가 실행되고 난 다음에는 'SUCCESS ~~~' 라는 메세지를 리턴할 수도 있지만, 실제로 수정, 생성, 삭제의 행위가 반영되고 난 다음의 최신화 된 값 혹은 리스트를 반환해주는 것이 프론트에서 재사용하기 효율적입니다! 이러한 것들은 지금 당장 수정이 필요하기보다는 내용만 가져가시고 추후에 리팩토링하며 수정하시면 될 것 같아요 :)